module1.js

```diff
@@ -53,7 +62,9 @@ function question3(number) {
 */
 
 function question4(array) {
-  // TODO:
+  for(var i = 0; array.length > i; i+2){
```

Comment:
`i + 2` means you're adding 2 to the variable `i`. But where are you assigning that result to? That isn't captured anywhere here